plasmapy/plasma/equilibria1d.py

```diff
@@ -0,0 +1,100 @@
+__all__ = ["HarrisSheet"]
+
+import astropy.constants as const
+import astropy.units as u
+import math
+import numpy as np
+from plasmapy.utils.decorators.validators import validate_quantities
+
+class HarrisSheet:
+
+    """
+    Define a Harris Sheet Equilibrium.
+
+    Parameters
+    ----------
+    B0 : `~astropy.units.Quantity` 
+         magnetic field.
+
+    delta : 'float'
+        Delta is the thickness of the sheet.
+
+    P0 : 'float'
+        Plasma Pressure.
+
+    Notes
+    -----
+    A current sheet is current limited to a surface.
+
+    Examples
+    --------
+    >>> harris_sheet = HarrisSheet(delta = 3*u.m ,B0 = 2*u.T)
+    >>> harris_sheet.magnetic_field(y = 5*u.m)
+    <Quantity 1.8622... T>
+
+    """
+
+    def __init__(self, B0, delta, P0=0 * u.Pa):
+        self.B0 = B0
+        self.delta = delta
+        self.P0 = P0
+
+    @validate_quantities
+    def magnetic_field(self, y: u.m) -> u.T:
+        r"""
+        Compute the magnetic field along y = 0.
```

Comment:
```suggestion
        Compute the magnetic field along :math:`y = 0`.
```
...so that this gets formatted as math.